app/models/claim.rb

```diff
@@ -197,7 +197,11 @@ def submittable?
     valid?(:submit) && !submitted? && submittable_email_details? && submittable_mobile_details?
   end
 
-  def approvable?
+  def approvable?(current_admin: nil)
+    if current_admin && high_risk_ol_idv? && !current_admin.is_service_admin?
```

Comment:
Fine for now, but I think all this logic (and the rest of the approvable? / rejectable?) logic should be in on the Decision / a DecisionForm.

Reply:
I agree that would be better. `Amendment` and those class methods too i think could do with some addressing